### DISCUSSION TITLE
ruan-cat-own-notes/50projects50days-vue3/01-expanding-cards/

### DISCUSSION THREAD

**Discussion by giscus** [RuanZhongNan/RPGMV-dev-notes#115] [Announcements]
# ruan-cat-own-notes/50projects50days-vue3/01-expanding-cards/

演示 demo 文章教程 https://juejin.cn/post/7262726113083752503#heading-3; B 站视频教程

https://www.ruan-cat.com/ruan-cat-own-notes/50projects50days-vue3/01-expanding-cards/

**Comment by ruan-cat** [comment]
假设这是我写的插件API。然后我评论。事实上到处都可以看得到。

**Reply by ruan-cat** [reply]
- 我写好文档了。
- 用户在文档下面评论。
- 我甚至可以收到邮件通知。
- 而且其他人还可以看到。

**Comment by ruan-cat** [comment]
找茬的阮喵喵：不是，这个怎么用啊？我都不会导入。呜呜呜。

**Reply by ruan-cat** [reply]
自问自答的阮喵喵：试着使用 pnpm i @ruan-cat/fuck-you 命令来安装依赖。
